andela/wger_asgards
# [#137085775] Manager deactivate trainers from gym.

#### What does this PR do?
Developed a way for manager to delete or deactivate gym users.
#### Description of Task to be completed?
Any gym manager can deactivate or delete the account of any user in his/her gym.
#### How should this be manually tested?
1. The manager must be logged in.
2. Under user list, a table of all users is shown.
3. For each user, there is a deactivate and delete links.
4. Deactivate only makes the user account inaccessible until the manager changes his mind.
5. Delete removes the user from the system permanent. A confirm form is given before the account is deleted. 
#### What are the relevant pivotal tracker stories?
#137085775
